src/ussd_registration.js

```diff
@@ -229,6 +229,19 @@ go.app = function() {
                 }
             });
         });
+// Function to check thatprefix is validated from list 
+        function phoneNumberPrefix(inputtxt)
+        {
+         var regexpression = new RegExp(/^(0701|0702|0703|0705|0706|0708|0806|0807|0808|0809|0810|0811|0812|0813|0814|0815|0816|0817|0818|0909|0908|0902|0903|0905|0906|0907)([0-9]{7})$/);
```

Comment:
If we can pull this from the config it would be great. This will allow us to add new prefixes without updating the code.

You can find the config for the train ussd app that you are changing here: 
https://github.com/praekelt/mama-ng-jsbox/blob/develop/test/train_ussd.test.js#L18

Reply:
I also see this function appears twice?